docs/data/data-grid/demo/DemoHub.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import EditingWithDatePickers from '../editing/EditingWithDatePickers';
+import ExcelExport from '../export/ExcelExport';
+import QuickFilteringGrid from '../filtering/QuickFilteringGrid';
+import BasicDetailPanels from '../master-detail/BasicDetailPanels';
+import RowGroupingInitialState from '../row-grouping/RowGroupingInitialState';
+import FullFeaturedDemo from './FullFeaturedDemo';
+import ControlPinnedColumns from '../column-pinning/ControlPinnedColumns';
+import ColumnVirtualizationGrid from '../virtualization/ColumnVirtualizationGrid';
+import CursorPaginationGrid from '../pagination/CursorPaginationGrid';
+import ColumnSelectorGrid from '../column-visibility/ColumnSelectorGrid';
+import RowOrderingGrid from '../rows/RowOrderingGrid';
+import TreeDataFullExample from '../tree-data/TreeDataFullExample';
+import RestoreStateInitialState from '../state/RestoreStateInitialState';
+
+export const featuresSet = [
+  {
+    id: 1,
+    name: 'Master detail',
+    description: 'A parent row, with a collapsible child panel',
+    plan: 'Pro',
+    detailPage: '/master-detail/',
+    demo: <BasicDetailPanels />,
+  },
+  {
+    id: 2,
+    name: 'Editing inline',
+    description: 'Edit your data inside the grid',
+    plan: 'Community',
+    detailPage: '/editing/',
+    demo: <EditingWithDatePickers />,
+  },
+  {
+    id: 3,
+    name: 'Pagination',
+    description: 'Easily paginate your rows and only fetch what you need',
+    plan: 'Community',
+    detailPage: '/pagination/',
+    demo: <CursorPaginationGrid />,
+  },
+  {
+    id: 4,
+    name: 'Save and Restore state',
+    description:
+      'Save and Restore internal state and configurations like active filters and sorting',
```

Comment:
Some features are named as "Foo Bar", others as "Foo bar". I'm favor of using the same casing for all.

```suggestion
    name: 'Save and restore state',
    description:
      'Save and restore internal state and configurations like active filters and sorting',
```